sbi/inference/snpe/snpe_base.py

```diff
@@ -435,6 +425,22 @@ def _train(
         self._summary["epochs"].append(epoch)
         self._summary["best_validation_log_probs"].append(self._best_val_log_prob)
 
+    def _calibration_kernel_loss(
+        self, round_idx, theta_batch, x_batch, masks
+    ) -> Tensor:
+        """Return round dependent log_prob multiplied by calibration kernel."""
```

Comment:
round*-*dependent, log prob (i.e. if it is log_prob it should be `log_prob`).

Maybe "Return the right calibrated loss/log_prob, depending on the round."?